frontend/src/middleware.ts

```diff
@@ -0,0 +1,36 @@
+import { NextResponse } from "next/server";
```

Comment:
What is the purpose of having this `middleware.ts` file for now? I see you are testing adding request headers, but you might want to clean up the code before merging into `master` if you are not going to use them.

Reply:
yes sure